python/understack-workflows/understack_workflows/main/sync_server.py

```diff
@@ -77,3 +77,8 @@ def main():
     response = client.update_node(node.uuid, patches)
     logger.info(f"Patching: {patches}")
     logger.info(f"Updated: {response}")
+    return ironic_node.provision_state
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Since the value is returned by the main function, how is it being captured by the container runner? Seems like you'd either have to grab it from stdout or from a file?

Reply:
Made changes to send the output to stdout.